azure-pipelines.yml

```diff
@@ -37,7 +37,7 @@ stages:
     - template: FhirToDataLake/azurepipelines/ci.yml
 
 - stage: OverallTest
-  condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/main'))
+  condition: succeeded()
```

Comment:
Always run this test? If multiple PRs are commited at same time, the test will fail due to the conflict.

Reply:
Yes. This is for preview branch only